File: backends/php-laravel/app/Domain/Notification/Entities/Notification.php
<?php

/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */

declare(strict_types=1);

namespace App\Domain\Notification\Entities;

use App\Support\Entities\AbstractEntity;

/**
 * 通知Entityクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\Domain\Notification\Entities
 */
class Notification extends AbstractEntity
{
    public ?int $id = null;
    public ?string $token = null;
}

File: backends/java-spring/src/main/java/com/authorization/domain/notification/entities/Notification.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.domain.notification.entities;

import com.authorization.support.entities.AbstractEntity;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * 通知Entityクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class Notification extends AbstractEntity {

    private Long id;
    private Long staffId;
    private Integer messageType;
    private String title;
    private String message;
    private String url;
    private boolean read;
}
